include/klee/Module/KModule.h

```diff
@@ -381,6 +381,8 @@ class KModule {
 
   KBlock *getKBlock(const llvm::BasicBlock *bb);
 
+  bool inMainModule(const llvm::Instruction &i);
```

Comment:
For convenience: for instance we meet a `br` instruction (or `call`, `ret`...) not in main module. Then we do not want to record it as it gives nothing to user in a report. Without this overload we would ask a parent for instruction, and call `inMainModule` on it. Overload added to avoid calls to `getParent`.